index.bs

```diff
@@ -6668,7 +6674,7 @@ In effect, where <var ignore>x</var> is a <emu-val>Number</emu-val> value,
 
     1.  Let |x| be [=?=] [=ToNumber=](|V|).
     1.  If |x| is <emu-val>NaN</emu-val>, +∞, or −∞,
-        then [=ECMAScript/throw=] a <emu-val>TypeError</emu-val>.
+        then [=ECMAScript/throw=] a {{TypeError}}.
```

Comment:
> Speaking of that, I'm not too clear on the difference between ES/throw and ES binding throw -- they seem to be equivalent IMO.

The first one is just a proxy to the one in the ES spec. The second one deals with DOMExceptions too.